frontend/src/components/activePage/ActiveView/ActiveView.tsx

```diff
@@ -38,12 +40,72 @@ const ActiveView: FC<IActiveViewProps> = ({ ...props }) => {
     <Col span={24} lg={22} xxl={20}>
       <Box
         header={{
-          center: !managerView && (
+          center: !managerView ? (
             <div className="h-full flex justify-center items-center px-5">
               <p className="md:text-2xl text-lg text-center mb-0">
                 <b>Active Instances</b>
               </p>
             </div>
+          ) : (
+            managerView &&
+            currentView === WorkspaceRole.manager && (
+              <div className="h-full flex justify-center items-center gap-4 px-5">
+                <Button
+                  className="hidden xl:block"
+                  type="primary"
+                  shape="round"
+                  size="middle"
+                  icon={<FullscreenOutlined />}
+                  onClick={() => {
+                    setExpandAll(true);
+                  }}
+                >
+                  Expand
+                </Button>
+                <Button
+                  className="hidden xl:block"
+                  type="dark"
+                  shape="round"
+                  size="middle"
+                  icon={<FullscreenExitOutlined />}
+                  onClick={() => {
+                    setCollapseAll(true);
+                  }}
```

Comment:
```suggestion
                  onClick={() => setCollapseAll(true)}}
```
Keep it compact, check other occurrences